bot/cogs/dm_comms.py

```diff
@@ -0,0 +1,48 @@
+from discord import DMChannel, Message
+from discord.ext import commands
+from rodhaj import Rodhaj
+
+
+class DmCommunications(commands.Cog):
+    def __init__(self, bot: Rodhaj):
+        self._bot = bot
+        self._pool = bot.pool
+
+    async def handle_user_no_thread(self, message: Message):
+        channel = message.channel
+        context = await self._bot.get_context(message)
```

Comment:
It would be more efficient just to construct the context once as you would need it in both cases anyways